addons/mrp_subcontracting/views/res_partner_views.xml

```diff
@@ -10,4 +10,15 @@
             </xpath>
         </field>
     </record>
+    <record id="view_partner_mrp_subcontracting_account_filter" model="ir.ui.view">
+        <field name="name">res.partner.select.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_res_partner_filter" />
+        <field name="groups_id" eval="[(4, ref('mrp.group_mrp_user'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//filter[@name='supplier']" position="after">
```

Comment:
@amoyaux If only mrp_subcontracting will be installed, this will throw an error of 'cannot be located in parent view' as account is not installed yet.